mailstore-connector/src/com/axonivy/connector/mailstore/MailStoreService.java

```diff
@@ -404,10 +404,10 @@ public void close() {
 					}
 				}
 			}
+			Thread.currentThread().setContextClassLoader(originalClassLoader);
 			if(exception != null) {
-				buildError("close").withCause(exception);
+				buildError("close").withCause(exception).throwError();

```

Comment:
die close funktion wird auch aufgerufen im catch() von eine anderen Methode. dann würde in der anderen methode noch mal ein try() { close(); } catch(Exception ex) { // silent ignore }

sonst hast Du eine exception im exception handling.

Reply:
Ok, habe ich geändert!